### PR TITLE
Use Miniswig from external submodule

### PR DESCRIPTION
**Pull request by Vankata453**
Adds Miniswig as an external submodule from [WindstilleTeam/miniswig](https://github.com/WindstilleTeam/miniswig), which is more actively maintained.

### CODE REVIEW

File: src/scripting/wrapper.hpp [hunk 1]
```diff
@@ -3,8 +3,8 @@
  *  'src/scripting/wrapper.interface.hpp'
  * DO NOT CHANGE
  */
-#ifndef HEADER_SUPERTUX_SCRIPTING_WRAPPER_HPP
-#define HEADER_SUPERTUX_SCRIPTING_WRAPPER_HPP
+#ifndef HEADER_SUPERTUX_SQUIRREL_WRAPPER_HPP
```
Comment (tobbi) [inline src/scripting/wrapper.hpp:6]: Why was this changed?

Comment (Vankata453) [reply src/scripting/wrapper.hpp:6]: "SCRIPTING" was inconsistent when using Miniswig in other projects. I settled for using "SQUIRREL" because it'd fit in all scenarios, but it could also be made so it's set by an argument.

Comment (Semphriss) [reply src/scripting/wrapper.hpp:6]: I thought the include guards were meant to follow the file's pathname?

Comment (Vankata453) [reply src/scripting/wrapper.hpp:6]: That's true. I could make it so it generates the header based on the path in `--output-hpp-include`.

**Review by mrkubax10 — APPROVED** [review]
